release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -144,6 +144,77 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
+
+==== CSI driver volume detach with non-graceful node shutdown (Technology Preview)
+Container Storage Interface (CSI) drivers can now automatically detach volumes when a node goes down non-gracefully. When a non-graceful node shutdown occurs, you can manually add an out-of-service taint on the node to allow volumes to automatically detach from the node. This feature is supported with Technology Preview status.
+
+For more information, see ???.
+
+==== VMware vSphere encryption support is generally available
+You can encrypt virtual machines (VMs) and persistent volumes (PVs) on {product-title} running on vSphere. This feature allows user-provisioned persistent storage to be encrypted over wire.
+
+For more information, see ???.
+
+==== VMware vSphere CSI topology support for multiple datacenters is generally available
+{product-title} 4.12 introduced the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. {product-title} 4.13 introduces support for deploying over multiple datacenters as well.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+==== Creating more than one default storage class is generally available
+{product-title} 4.13 allows you create more than one default storage class. This feature makes it easier to change the default storage class because you can create a second storage classes defined as the default, thus temporarily having two default storage classes, before removing default status from the previous default storage class.
+
+While it is acceptable to have multiple default storage classes for a short time, you should ensure that eventually only one default storage class exists.
+
+For more information, see ???.
+
+==== Managing the default storage class is generally available 
+{product-title} 4.13 introduces the `spec.storageClassState` field in the ClusterCSIDriver object, which allows you to manage the default storage class to accomplish several different objectives:
+
+* Enforcing static provisioning by disabling dynamic provisioning.
+
+* When you have other preferred storage classes, preventing the storage operator from re-creating the initial default storage class.
+
+* Renaming, or otherwise changing, the default storage class
+
+For more information, see ???.
+
+==== Retroactive default StorageClass assignment (Technology Preview)
+There are some cases where the default storage classes are not assigned, or more than one default is defined. This feature allows you to retroactively assign a default storage class to correct theses situations. This feature is supported with Technology Preview status.
```

Comment:
s/theses/these 

(I hate those meeces to pieces!)

Reply:
Hee hee.